indico/modules/events/forms.py

```diff
@@ -102,3 +103,12 @@ class LectureCreationForm(EventCreationFormBase):
     person_link_data = EventPersonLinkListField(_('Speakers'))
     description = TextAreaField(_('Description'), widget=CKEditorWidget())
     theme = IndicoThemeSelectField(_('Theme'), event_type=EventType.lecture, allow_default=True)
+
+
+class UnlistedEventsForm(IndicoForm):
+    enabled = BooleanField(_('Enabled'), widget=SwitchWidget(), default=False)
+    restricted = BooleanField(_('Require permission'), [HiddenUnless('enabled', preserve_data=True)],
```

Comment:
How about "Restricted" or "Restrict creation"? And maybe a description like "Restrict creation of unlisted events to the authorized users below"